Sources/Logging/Logging.swift

```diff
@@ -256,3 +268,41 @@ private extension NSLock {
         return body()
     }
 }
+
+extension Logging.MetadataValue: ExpressibleByStringLiteral {
+    public typealias StringLiteralType = String
+
+    public init(stringLiteral value: String) {
+        self = .string(value)
+    }
+}
+
+extension Logging.MetadataValue: ExpressibleByStringInterpolation {
+    #if !swift(>=5.0)
+    public init(stringInterpolation strings: Logging.MetadataValue...) {
+        let s = strings.reduce("", { if case .string(let s) = $1 { return $0 + s } else { return $0 + "<>" }})
```

Comment:
why the <>?

Reply:
@tomerd in Swift 5 the whole String interpolation thing got improved a lot and the conformance can be automatically synthesised automatically. I just put the code in there to make it compile with Swift 4.2 and didn't put much effort in it because I think the package will become Swift 5 only anyway but for now it makes it usable with 4.2. I can improve it if you want

Reply:
done